tests/resources/mlflow-test-plugin/setup.py

```diff
@@ -8,7 +10,7 @@
     packages=find_packages(),
     # Require MLflow as a dependency of the plugin, so that plugin users can simply install
     # the plugin & then immediately use it with MLflow
-    install_requires=["mlflow"],
+    install_requires=["mlflow @ file://{}".format(ROOT_DIR)],
     entry_points={
         # Define a Tracking Store plugin for tracking URIs with scheme 'file-plugin'
         "mlflow.tracking_store": "file-plugin=mlflow_test_plugin.file_store:PluginFileStore",
```

Comment:
For skinny client tests removing the mlflow dependency worked. It does not mess with the tests since it is implied mlflow is installed.

It allows for more flexibility in the testing approach

Reply:
Makes sense. @smurching what do you think?